src/orqa.cpp

```diff
@@ -0,0 +1,71 @@
+#include "orqa.h"
+
+GENERIC_CRC8 ghst_crc(GHST_CRC_POLY);
+
+Orqa::Orqa()
+{
+}
+
+void Orqa::Init()
+{
+    delay(VRX_BOOT_DELAY);
```

Comment:
Does the Orqa also need the 7s boot delay? Seems suspiciously like this was copied from the HDZ one?
May need to be higher / lower?

Reply:
Yeah I've heard the Orqa's take a while to boot.
The intent for the boot delay is to wait long enough for the module to boot, so that when the backpack first polls out to the handset and gets the current channel, the VRX module is in a state where it can actually accept the change.

Reply:
If you change the channel while goggles are powered off, and hit "Send" in the LUA, then power the goggles on, does the new channel update on boot?

Reply:
@jonas-koeritz any updates on the correct boot delay timing? @JyeSmith tested this and indicated that 7s is not long enough to pick up channel changes on boot. I am planning on cutting a new release of the Backpack firmware soon (for the new skyzone target), so if you are able to fix this before then, this could get included too.

Reply:
I did some more testing and it looks ok.  I think the first rx I was using as a backpack just had terrible wifi range.